scripts/json/nodes-upload.json

```diff
@@ -0,0 +1,44 @@
+{
+   "host_infos": [
+      {
+         "local_ip": "30.213.43.10",
+         "node_id": "e109465c-5ed6-400e-806d-fbefd12f94a9",
+         "mac_address": "D1:A0:C9:34:C8:29",
+         "node_name": "upl009030",
+         "server_port": 50001,
+         "veth": "eth0"
+      },
```

Comment:
Somehow pkommoju:nmm_ncm_api_07 branch is not in sync with the master, for example, node-upload.json file is an existing file in alcor:master branch. https://github.com/futurewei-cloud/alcor/blob/master/scripts/json/nodes-upload.json

Please fix your branch before calling for a review. The same issue repeatedly show up for 3 or 4 PRs. @pkommoju